### PR TITLE
removing accession so the rolls argument is what is accessed

### LINKED ISSUE #395: Refactor defineRollDataGetter
We are changing the data structure of roll tables from

```js
setup.smithy.rollData: {
    activity: [
        [80, 'extremely busy'],
        [70, 'very busy'],
        [60, 'rather busy'],
        [50, 'reasonably busy'],
        [40, 'not terribly busy'],
        [30, 'reasonably quiet'],
        [20, 'rather quiet'],
        [10, 'totally empty'],
        [0, 'totally empty']
        ]
}
```

to 
```js
setup.smithy.rollData: {
activity: {
      description: 'How busy is the blacksmith? Is there a backlog a mile long, or are apprentices sitting idle?',
      preceding: 'Smithy:',
      rolls: [
        [80, 'extremely busy'],
        [70, 'very busy'],
        [60, 'rather busy'],
        [50, 'reasonably busy'],
        [40, 'not terribly busy'],
        [30, 'reasonably quiet'],
        [20, 'rather quiet'],
        [10, 'totally empty'],
        [0, 'totally empty']
      ]
    },
```

Siloing the rolls allows us to add properties to the roll, such as the description (to allow editing of rolls via #258), whether the roll should be re-rolled upon subsequent visits (for cases such as smithy activity), and plenty of other things.

This means that the pathing of `defineRollDataGetter` needs to change and reflect this. At the moment, the `rolls` argument anticipates being one level "down" from the array, with the key to access the array being provided as the `keyName` argument. This could be simplified; since the function only handles one roll, there's no reason for it to need to access the other properties (it doesn't need to know what the roll array for cleanliness is if it's dealing with wealth).

### PR DESCRIPTION
**Pull request by ryceg**
## What does this do?

Refactors the defineRollDataGetter to pass the nested object directly

## How was this tested? Did you test the changes in the compiled `.html` file?

Yes

## Is there a [GitHub Issue](https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/issues?q=is%3Aopen+is%3Aissue) that this is resolving?

Closes #395

## We are shifting from a GPL3 license to MIT. Are you okay with us relicensing your code contributions under this new license?

Yes

### CODE REVIEW

File: lib/alchemist/alchemistData.ts [hunk 1]
```diff
@@ -5,10 +5,42 @@ import { random } from '../src/random'
...
   rollData: {
-    wealth: ThresholdTable
-    size: ThresholdTable
-    cleanliness: ThresholdTable
-    expertise: ThresholdTable
+    wealth: {
+      description: string
+      preceding: string
+      rolls: ThresholdTable
+    }
```
Comment (Glinkis) [inline lib/alchemist/alchemistData.ts:12]: Might be nice to extract an interface for the new signature.

**Review by Glinkis — APPROVED** [review]
Looks good to me.

**Comment by Glinkis** [comment]
You'll need to address the build failures though.

**Comment by ryceg** [comment]
The build failure is just the test, but I lack the understanding to troubleshoot why simply adding it in isn't fixing the issue.

**Pull request merged**